perlgov: proposed edits for election process

rjbs opened the pull request:
( to be discussed on perl-core )

rjbs committed:
perlgov: allow deferral of Special Election

When, for example, there is a need for Special Election near the
date of an upcoming Term Election.

rjbs committed:
perlgov: handling uncontested elections

rjbs committed:
perlgov: allow for no vote when 2-seats, 2-candidates

rjbs committed:
perlgov: limit the time period for objection to deferral

rjbs committed:
perlgov: allow PSC to cancel election deferment

rjbs committed:
perlgov: clarify time frame for newly Term Election

hvds commented on pod/perlgov.pod:
```diff
@@ -395,8 +395,8 @@ entire Steering Council is ever vacant, a Term Election will be held instead.
-Core Team member objects, the Special Election must be organized within one
-week.
+Core Team member objects within one week, the Special Election must be
+organized within one week.
```
Given that without deferment the vote must be organized within two weeks, I would have expected an objection to deferment to cause it to fall back to two weeks rather than a tighter one-week schedule.

rjbs replied:
This is fine with me and I have no feelings about it either way.

jkeenan commented:
Is there anything in this proposed new document that is at variance with @xdg's https://github.com/Perl/perl5/pull/18887?

rjbs committed:
perlgov: allow two weeks to organize after objection

This change suggested by Hugo in review.

karenetheridge commented:
LGTM. I see nothing controversial here.

xdg commented on pod/perlgov.pod:
```diff
@@ -427,7 +433,9 @@ anonymously.  After voting is complete, candidates are ranked using the
 occurs, it may be resolved by mutual agreement among the tied candidates, or
 else the tie will be resolved through random selection by the Vote
-Administrator.
+Administrator.  Unless there are more candidates than open seats, no vote is
```
As with code, I find "unless" harder to mentally parse than an "if".  Also, the rest of the document uses the term "vacancy", not "open seats" so I suggest that be changed for consistently.  I have a mild preference that we call this out either at the top of item 2 or as a new item 2 so that this check appears between "nomination" and "ranking" in the document in the same way it will happen in the actual process.

rjbs committed:
perlgov: slight reorganization of "no contest" text

rjbs committed:
perlgov: replace "administer" with "organize"

Other parts of perlgov forbid PSC members from being vote
administrators, so using "administer" here was confusing.

nwc10 commented:
This PR should be closed, because the branch is out of date?

(The first conflict I found was different text for *perlgov: allow deferral of Special Election*)

rjbs commented:
Yes, thanks.   I must not have pushed to this branch on GitHub before merging to blead!
